app/views/sessions/payment.html.erb

```diff
@@ -5,7 +5,8 @@
 
 
 <h1>Payment</h1>
```

Comment:
seems like we need to find a way to update the pricing and payment pages properly